libgenic/power.h

```diff
@@ -14,10 +16,31 @@ struct power_params
     double PrimordialIndex;
 };
 
+/*A note on gauge choice:
+ * We run Gadget in N-body gauge (Fidler et al 2015) arxiv:1505.04756.
+ * See also (Valkenburg & Villaescusa-Navarro 2016) arxiv:1610.08501, which includes
+ * an N-body IC code.
+ *
+ * In this gauge, as long as there is zero anisotropic stress (true if omega_r = 0)
+ * the equations of motion are the Newtonian equations and the density is the
+ * comoving synchronous gauge density. Roughly speaking one is choosing a spatial gauge
+ * comoves with the large-scale radiation perturbation, making them zero.
+ *
+ * The velocities for this gauge can be obtained by differentiating the synchronous gauge
+ * velocity perturbations. Note that on super-horizon scales this is not the same as the
```

Comment:
You sure it is velocity perturbations? It appears we are diffing against density perturbations.